homework/generate-sequence/generateSequence.hpp

```diff
@@ -2,6 +2,14 @@
 #include <vector>
 
 std::vector<int> generateSequence(int count, int step) {
-    // TODO: Implement me :)
-    return {};
+
+    std::vector<int> sequence;
+
+    for (int counter = 0; counter < count; counter++)
+    {
+        auto element = sequence.empty() ? step : sequence.back() + step;
```

Comment:
You always check this. I this is better to increment step, instead checking if sth was added to vec. More clear and simple.